_posts/2017-12-18-gentle-intro-nsec-badge.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Gentle Introduction to the NorthSec Badge
+tags: electronics arm hacking
```

Comment:
See topics. Remove `electronics` add `hardware`, `tutorial` and others if you think they fit.